### DISCUSSION TITLE
When Strawberry Music Player going to be available for Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by SuperSentient** [strawberrymusicplayer/strawberry#981] [General]
Ubuntu 22.04 came out this April and its June now. So when can I expect the Strawberry Music Player to be available for Ubuntu 22.04?

**Comment by jonaski** [comment]
https://files.strawberrymusicplayer.org/strawberry_1.0.5-jammy_amd64.deb

**Reply by SuperSentient** [reply]
Thanks I see you updated the website as well. :)
